themes/gokarna/layouts/partials/post.html

```diff
@@ -2,6 +2,7 @@
 
     <div class="post-header-section">
         <h1>{{ .Title }}</h1>
+        <small role="doc-subtitle">This is a subheading</small>
```

Comment:
@yashmehrotra Please add templating as you see fit

Reply:
You just have to put `{{ .Description }}`

Reply:
Updated